templates/showforum.tpl

```diff
@@ -12,6 +12,12 @@
   | <a href="#post"><b>Post New Thread</b></a>
   | <a href="#" id="view-all-images">View All Images</a>
   | <a href="#" id="night-mode">Night Mode: <span id="night-mode-status">Off</span></a>
+  <div style="display: inline"> | </div>
+<<<<<<< d37bde8b21c7f9263567902ecdd2f459138e51f6
+  <a href="#" id="admin-mode">Night Mode: <span id="admin-mode-status">Disabled</span></a>
+=======
+  <a href="#" id="admin-mode">Admin Mode: <span id="admin-mode-status">Disabled</span></a>
+>>>>>>> Disable Admin Feature
```

Comment:
:)

Reply:
Some day I'll learn how to read diffs the first time.